iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -12,9 +12,8 @@ import CoreData
 
 class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, StoreSubscriber {
     @IBOutlet weak var tableView: UITableView!
-    var articles: [MBArticle] = []
-    var attributedTitles: [NSAttributedString] = []
```

Comment:
I just compute these on the fly now in the call to configure. Didn't want to deal with a data structure for these since articles is no longer a simple list. Also didn't want to add an extension on the model objects for fear of core data (but maybe that'd be fine)